website/oauth/views.py

```diff
@@ -54,7 +54,7 @@ def oauth_callback(service_name, auth):
             _forward_to_addon_service('oauth1')
         elif provider._oauth_version == 2:
             _forward_to_addon_service('oauth2')
-        return {}
+        return {'enable_gv': True}
```

Comment:
Should this return the actual value of the waffle flag?

Reply:
It could, but on line 52 it is already checking if the flag is active, so at this point the waffle flag value would always be `True`.